### ISSUE TITLE
SubmitHandler type error

### ISSUE THREAD

**Issue by lucasraziel** [Rocketseat/unform#161]
**Description of bug**

Error when creating a submit function for the form:

Type 'SubmitHandler<DataContent>' is not assignable to type 'SubmitHandler<FormContent>'.
  Type 'FormContent' is missing the following properties from type 'DataContent': email, passwordts(2322)

**To Reproduce**

Create a page with Form Element and SubmitHandler function in the same file:

Example:
```
...
const handleSubmit:SubmitHandler<DataContent> = (data) => {
    dispatch(signInRequest(data));
  };

  return (
    <>
         <Form onSubmit={handleSubmit}>
...

```

**Expected behavior**

It shouldn't raise any error

**Exception or Error**

Type 'SubmitHandler<DataContent>' is not assignable to type 'SubmitHandler<FormContent>'.
  Type 'FormContent' is missing the following properties from type 'DataContent': email, password  TS2322

**Environment:**

O.S Ubuntu 19.0.4
Node v 12.13

**Additional context**

It worked when I changed the Form.tsx to:

```
export interface FormProps<T> extends Omit<HTMLFormProps, 'onSubmit'> {
    initialData?: object;
    children: React.ReactNode;
    context?: Context;
    schema?: ObjectSchema<object>;
    onSubmit: SubmitHandler<T>;
}
```

**Comment by eduardoborges** [comment]
I found this error too.

**Comment by felipecoders** [comment]
i found this error too.

**Comment by diego3g** [comment]
Unform released 2.0 🎉

And we fixed that in this release :)

Check the [migration guide](https://unform.dev/migration-guide).
